evaluate_topic_modelling.py

```diff
@@ -72,6 +72,52 @@ def calculate_matching_score(self, annotator_topics, generated_topics):
         return np.mean(match_scores)
 
 
+class CrossEncoderMetric1to1(Metric):
+    name = "cross-encoder/nli-deberta-v3-base - 1 to 1 matching."
+
+    def __init__(self):
+        self.ce = CrossEncoder('cross-encoder/stsb-TinyBERT-L-4')
+
+    def calculate_matching_score(self, annotator_topics, generated_topics) -> float:
+        scores = []
+        for annotator_topic in annotator_topics:
+            anotation_scores = []
+            for generated_topic in generated_topics:
+                anotation_score = self.ce.predict([(annotator_topic, generated_topic)]).tolist()[0]
```

Comment:
nevypisuje to warning ze nemas delat .predict v loopu? na grafarne to proste vypocita vsechno najednou, kdyz mu das hromadu párů, ale asi to je jedno v nasem pripade, teda hlavne dokud delame evaluaci 10 prvků

Reply:
Nevypisovalo, nedelal jsem to na gpu tho, ale jo, muzu to predelat, tak jako tak nejakym iterovanim budu muset vytvorit pary